test/run-verify-host.sh

```diff
@@ -18,19 +18,11 @@ nproc
 free -h
 rpm -q cockpit-system
 
-# install browser; on RHEL, use chromium from epel
-# HACK: chromium-headless ought to be enough, but version 88 has a crash: https://bugs.chromium.org/p/chromium/issues/detail?id=1170634
-if ! rpm -q chromium; then
-    if grep -q 'ID=.*rhel' /etc/os-release; then
-        # There is no EPEL for RHEL 9 yet, force 8
-        # RHEL 9 has a broken stub epel.repo
-        rm -f /etc/yum.repos.d/epel.repo
-        dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-        sed -i 's/$releasever/8/' /etc/yum.repos.d/epel*.repo
-        dnf config-manager --enable epel
-    fi
-    dnf install -y chromium
-fi
+# install browser; on RHEL, use firefox-nightly as chromium broke CDP on rhel-9
+# Install firefox to pull in all the deps
+dnf -y install firefox
```

Comment:
Why do we install firefox from the repos and then install it again from mozilla.org?

Reply:
"Install firefox to pull in all the deps"

We need some libraries to be present on the system - therefore I install firefox from repos so it pulls it in. But we need nightly as that only has CDP.

Reply:
Ah, makes sense!